content/page/obf-bosc-collaborationfest-2025.md

```diff
@@ -0,0 +1,27 @@
+---
+author: hmenager
+cover:
+  alt: CoFest-2022
+  image: https://www.open-bio.org/wp-content/uploads/2022/08/image4.jpg
+date: "2025-02-16T09:35:44+00:00" #creation date
+guid: https://www.open-bio.org/?page_id=7453
+title: ISMB CollaborationFest 2025
+url: /events/bosc-2025/ismb-collaborationfest-2025/
+bosc: yes
+---
+
+In conjunction with our annual meeting, [BOSC](/events/bosc/), the [OBF](/) will this year run a joint collaborative event organised by volunteers from four different COSIs: BOSC, Bio-Ontologies and Knowledge Representation (BOKR), Function COSI, and 3DSIG. At these events, participants work together to contribute code, documentation, training materials, and challenging analysis problems and use cases. If you are interested in learning and contributing in an intensely collaborative environment, then CollaborationFest is for you.
+
+BOSC has organized CollaborationFests (aka CoFests) every year before or after ISMB since 2010. (See, for example, [the 2024 CoFest](https://www.open-bio.org/events/bosc-2024/obf-bosc-collaborationfest-2024/).) This year, we are excited to hold CollaborationFest as part of ISMB/ECCB, opening it to all registered ISMB/ECCB participants.
+
+Due to space constraints, participation in CollaborationFest is limited. To indicate your interest, [register for ISMB/ECCB 2025](https://www.iscb.org/ismbeccb2025/register) by June 20 and sign up for CollaborationFest as an “add-on” during the registration process. We will contact you by June 23 to gather additional information to help organize the event. Last-minute walk-ins may be possible if space allows.
+
+Limited remote participation will be possible, but engagement will depend on the people and projects active on the day. A video feed from the room will be available, but joining will require you to be a registered (virtual or in-person) ISMB attendee.
+
+### Date
+
+Wednesday July 25 to Thursday July 26.
+
+### Code of Conduct
+
+This event is under the [OBF Code of Conduct](/code-of-conduct/).
```

Comment:
It will probably have to follow BOSC and be under the ISCB's rather light-weight COC instead. In past years with the CoFest outside of ISMB, we could use our own COC.